### PR TITLE
feat: Add Datadog as an exporter for OTLP traces

### PR DESCRIPTION
**Pull request by yquansah**
Add documentation for using Datadog as a potential exporter for `OTLP` traces.

Completes FLI-581

### CODE REVIEW

File: examples/tracing/otlp/README.md [hunk 1]
```diff
@@ -32,3 +32,24 @@ To run this example application you'll need:
 1. Select `serviceName=flipt` from the search box
 1. Click 'Run Query'
 1. You should see a list of traces to explore
+
+### Datadog UI
+
+!['Datadog Example'](../../images/datadog.png)
+
+For exporting traces from OpenTelemetry to Datadog you have to provide this piece of configuration in the `otel-collector-config.yaml`:
```
Comment (markphelps) [inline examples/tracing/otlp/README.md:40]:
```suggestion
For exporting traces from [OpenTelemetry to Datadog](https://docs.datadoghq.com/opentelemetry/otel_collector_datadog_exporter) you have to configure the exporter in the `otel-collector-config.yaml`:
```

File: examples/tracing/otlp/README.md [hunk 2]
```diff
@@ -32,3 +32,24 @@ To run this example application you'll need:
...
+exporters:
+  datadog:
+    api:
+      site: datadoghq.com
+      key: ${DD_API_KEY}
+```
+
+**Note:** The `DD_API_KEY` should be replaced with your actual api key from Datadog.
+
+You can do this under the extensions configuration. You also have to add `datadog` as an entry in `exporters` under `service.pipelines.traces.exporters`.
```
Comment (markphelps) [inline examples/tracing/otlp/README.md:52]: do what under the extensions configuration?

Comment (markphelps) [inline examples/tracing/otlp/README.md:52]: You can do this under the extensions configuration. You also have to add `datadog` as an entry in `exporters` under `service.pipelines.traces.exporters`. 

For example:

```
service:
  extensions: [pprof, zpages, health_check]
  pipelines:
    traces:
      receivers: [otlp]
      processors: [batch]
      exporters: [logging, datadog]
```

**Review by markphelps — COMMENTED** [review]
one question, couple suggestions

File: examples/tracing/otlp/README.md (hunk 2 above)
Comment (yquansah) [reply examples/tracing/otlp/README.md]: @markphelps Simplified the wording on this.

**Review by markphelps — APPROVED** [review]
Lgtm!

**Review by GeorgeMac — APPROVED** [review]
Nice, that was a much more simple experience :D

**Pull request merged**